src/inmanta/deploy/state.py

```diff
@@ -62,12 +62,14 @@ class Compliance(StrEnum):
         based on a deploy attempt / compliance check for that intent.
     UNDEFINED: The resource status is undefined, because it has an unknown attribute. i.e. undefined status <=> undefined
         intent.
+    DEPLOYING: The resource is currently in the middle of a deploy attempt.
     """
 
     COMPLIANT = enum.auto()
     HAS_UPDATE = enum.auto()
     NON_COMPLIANT = enum.auto()
     UNDEFINED = enum.auto()
+    DEPLOYING = enum.auto()
```

Comment:
I don't know if this is intended to go into this state machine? @sanderr may know

Reply:
After discussion I removed this